### ISSUE TITLE
Encounter rate is MUCH lower with v0.0.4

### ISSUE THREAD

**Issue by PassiveModding** [AnthonyOSX/GO-Bot#67]
Encounters have gone down to 1 every 5-10 minutes rather than every 30 or so seconds

**Comment by dgol1115** [comment]
have your updoot. at least i know it's not just me.

**Comment by Majamulli** [comment]
Just had it run for ~30 minutes and there was no encounter at all. Stuck in a loop.

**Comment by PassiveModding** [comment]
yeah I started using v0.0.3 again just because I actually get encounters there
